src/nrniv/bgpdma.cpp

```diff
@@ -35,34 +35,10 @@ extern void vector_resize(IvocVect*, int);
 } // extern "C"
 extern void (*nrntimeout_call)();
 
-
-
-// The initial idea behind TWOPHASE is to avoid the large overhead of
-// initiating a send of the up to 10k list of target hosts when a cell fires.
-// I.e. when there are a small number of cells on a processor, this causes
-// load balance problems.
-// Load balance shuld be better if the send is distributed to a much smaller
-// set of targets, which, when they receive the spike, pass it on to a neighbor
-// set. A non-exclusive alternative to this is the use of RECORD_REPLAY
-// which give a very fast initiation but we have not been able to get that
-// to complete in the sense of all the targets receiving their spikes before
-// the conservation step.
-// We expect that TWOPHASE will work best in combination with ENQUEUE=2
-// which has the greatest amount of overlap between computation
-// and communication.
-// Note: the old implementation assumed that input PreSyn did not need
-// a BGP_DMASend pointer so used the PreSyn.bgp.srchost_ element to
-// help figure out the target_hosts_ list for the output PreSyn.bgp.dma_send_.
-// Since bgp.srchost_ is used only for setup, it can be overwritten at phase2
-// setup time with a bgp.dma_send_ so as to pass on the spike to the
-// phase2 list of target hosts.
-
```

Comment:
On second thought, it seems to me this comment contains valuable information and should be retained with just a replacement of TWOPHASE with use_phase2_ and removal of mention of RECORD_REPLAY

Reply:
Done ;)